File: Praktikum C/PRAK602-2210817210033-HariOctavianDelrossi.c
#include <stdio.h>
int main(){
    int kolom,i;
    scanf("%d",&kolom);
    int matriks[kolom];
    for(i=0;i<kolom;i++){
        scanf("%d",&matriks[i]);
    }
    for(i=0;i<kolom;i++){
        printf("%d ",matriks[i]*(i+1));
    }
    return 0;
}

File: Praktikum Python/PRAK602-2210817210033-HariOctavianDelrossi.py
kolom = int(input())
angka = list(map(int, input().split()))
matriks = []
a = 0
matriks.append(angka[a:a+kolom])
print()
for i in range(0, kolom):
    print(matriks[0][i]*(i+ 1), end=' ')